doc/src/build/programming-with-objects/ch5-dynamic-fields.md

```diff
@@ -12,7 +12,6 @@ Fortunately, Sui provides *dynamic fields* with arbitrary names (not just identi
 
 ### Current Limitations
 There are some aspects of dynamic fields that are not yet behaving as designed in this early release. We are actively working on these areas, but watch out for: - `remove` for dynamic fields not giving a full storage refund.
-- The lack of an `exists_` API for `dynamic_field` to check whether a field with a particular name is already defined on an object.
```

Comment:
Hi @ldback-aftermath, thanks for submitting this PR!

We do have `dynamic_field::exists_with_type`, and we have `dynamic_object_field::exists_`, but not `dynamic_field::exists_`, that's what this bullet point is referring to, if that makes sense.

The underlying reason for this is no longer valid, so we could implement it, but we haven't yet, so the limitation does still exist (i.e. we shouldn't remove this bullet)

If a clearer way to explain to this comes to you, please feel free to update this PR, otherwise feel free to close!